centreon/packages/ui/src/Form/Inputs/index.tsx

```diff
@@ -61,7 +63,15 @@ export const getInput = R.cond<
     R.always(FieldsTable)
   ],
   [R.equals(InputType.Grid) as (b: InputType) => boolean, R.always(Grid)],
-  [R.equals(InputType.Custom) as (b: InputType) => boolean, R.always(Custom)]
+  [R.equals(InputType.Custom) as (b: InputType) => boolean, R.always(Custom)],
+  [
+    R.equals(InputType.Checkbox) as (b: InputType) => boolean,
+    R.always(Checkbox)
+  ],
+  [
+    R.equals(InputType.MultiCheckbox) as (b: InputType) => boolean,
+    R.always(MultiCheckbox)
```

Comment:
I don't understand why there are two components for Checkbox. Can you explain me?

Reply:
As initial data, for a multi-checkbox, I have an array of options, such as: options = ["ok", "down", ...]. However, for a single checkbox, I have an object where I can include an Icon or not, like this: {label: "Email", Icon: EmailIcon}.

Reply:
I can use a multi-checkbox component for a single checkbox by having an array with a single element, like this: ["Include..."]. However, if I want to include an icon with the checkbox, I will need to use another component specifically designed for checkboxes with icons.